plasma_framework/contracts/src/transactions/WireTransaction.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.5.0;
+
+import "../utils/RLP.sol";
+
+/**
+ * @title WireTransaction
+ * @dev Utility functions for working with transactions in wire format.
+ */
+library WireTransaction {
+
+    using RLP for bytes;
+    using RLP for RLP.RLPItem;
+
+    struct Output {
+        uint256 amount;
+        bytes32 outputGuard;
+        address token;
+    }
+
+    /**
+    * @notice Returns output for transaction in wire format.
```

Comment:
I would add the assumption of using this function in @dev here.
Like the `outputs` must be a field on the second index and should be a list which is the format of {amount, outputGuard, token} accordingly.

Reply:
done